raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -309,3 +309,41 @@ def get_global_causal_policy(self, post_data):
                     "inner error: {}".format(e_str),
                 WidgetRequestResponseConstants.data: []
             }
+
+    def get_object_detection_metrics(self, post_data):
+        """Flask endpoint function to get Model Overview metrics
+        for the Object Detection scenario.
+
+        :param post_data: List of inputs in the order
+        [true_y, predicted_y, aggregate_method, class_name, iou_thresh].
+        :type post_data: List
+
+        :return: JSON/dict data response
+        :rtype: Dict[str, List]
+        """
+        try:
+            true_y = post_data[0]
+            predicted_y = post_data[1]
+            aggregate_method = post_data[2]
+            class_name = post_data[3]
+            iou_thresh = post_data[4]
+            exp = self._analysis.compute_object_detection_metrics(
+                true_y,
+                predicted_y,
+                aggregate_method,
+                class_name,
+                iou_thresh
+            )
+            return {
+                WidgetRequestResponseConstants.data: exp
+            }
+        except Exception as e:
+            print(e)
+            traceback.print_exc()
+            e_str = _format_exception(e)
+            return {
+                WidgetRequestResponseConstants.error:
+                    "Failed to get OD MO metrics,"
```

Comment:
"Failed to get objection detection model overview metrics,"  -- this will be shown to customers so let's avoid abbreviations